lib/Elastica/Query.php

```diff
@@ -330,7 +330,8 @@ public function toArray()
             $this->setQuery(new MatchAll());
         }
 
-        if (isset($this->_params['post_filter']) && 0 === count($this->_params['post_filter'])) {
+        $a = $this->_params['post_filter'];
```

Comment:
Nit picking but could we name this variable differently, something more describtive? I'm curious what happens on the next line in case `toArray()` does not on the varialbe?